cmd/server-main.go

```diff
@@ -205,6 +205,12 @@ var ServerFlags = []cli.Flag{
 		EnvVar: "MINIO_LOG_SIZE",
 		Hidden: true,
 	},
+	cli.BoolFlag{
+		Name:   "log-compress",
+		Usage:  "specify if we want the rotated logs to be gzip compressed or not",
+		EnvVar: "MINIO_LOG_COMPRESS",
+		Hidden: true,
+	},
```

Comment:
hidden flags on purpose to allow them to standardize for a bit, we may potentially change it.

Reply:
We can instead have a flag, say `log-compress-algo` a `cli.StringFlag`, which takes the compression algorithm as input, e.g `gzip` initially and more if we choose to support  other algorithms like `zstd`, `bz2`, etc?

Reply:
No I want to keep it gzip and be done. It is not worth adding all others..